packages/cli/src/swagger/specGenerator2.ts

```diff
@@ -14,13 +14,13 @@ export class SpecGenerator2 extends SpecGenerator {
   public GetSpec() {
     let spec: Swagger.Spec2 = {
       basePath: normalisePath(this.config.basePath as string, '/', undefined, false),
-      consumes: ['application/json'],
+      consumes: [DEFAULT_REQUEST_MEDIA_TYPE],
       definitions: this.buildDefinitions(),
       info: {
         title: '',
       },
       paths: this.buildPaths(),
-      produces: ['application/json'],
+      produces: [DEFAULT_RESPONSE_MEDIA_TYPE], // todo: Tsoa.Response.produces
```

Comment:
> A list of MIME types the APIs can produce.
This is global to all APIs but can be overridden on specific API calls.
Value MUST be as described under Mime Types.

Maybe I'm interpreting too much, but we should be able to leave this as is, remove the todo and override on path.method.produces?